source/developers-guide/coding-standards/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Coding Standards
+github_link: developers-guide/coding-standards/index.md
+indexed: true
+---
+
+The shopware team uses the following coding standards. All contributions to Shopware as well as plugins and customizations should also use these standards. 
+
+## PHP Coding Standard
+For PHP Code all contributions should use the [PSR-1: Basic Coding Standard](http://www.php-fig.org/psr/psr-1/) and [PSR-2: Coding Style Guide](http://www.php-fig.org/psr/psr-2/).
+
+## JavaScript Coding Standard
+Please see our dedicated page for the [JavaScript Coding Standard](/designers-guide/javascript-coding-style/).
```

Comment:
Fair enough. But the `{{ site.url }}` Twig tag is not compatible with the markdown link syntax so I will leave it for now.
If we ever have to move the devdocs to a subdirectory we have to change a whole lot of links anyway (regex ahead).